src/index.scss

```diff
@@ -1,3 +1,12 @@
+body {
+  margin: 0;
+}
+
 #root {
-  margin: 20px;
+  display: grid;
+  min-height: 100dvh;
+}
+
+.container {
+  padding: 30px;
```

Comment:
There was also an 8px default body margin in Chrome, probably something similar in other browsers too. The left/right spacing is only 2px larger than before when you actually render, but I'd consider taking it down to 24px/1.5em.

The bottom needs more space than you think it does because the swatch crows have negative margin on the bottom and it looks too cramped when you squish it close to the footer.

Reply:
The thing I don't like is that the space above and below the footer is so much

Reply:
I mean the footer padding